src/plugins/kibana_usage_collection/server/collectors/application_usage/README.md

```diff
@@ -10,7 +10,7 @@ To track a sub view inside your application (ie a flyout, a tab, form step, etc)
 
 #### For a React Component
 
-For tracking an application view rendered using react the simplest way is to wrap your component with the `TrackApplicationView` Higher order component:
+For tracking an application view rendered using react the simplest way is to wrap your component with the `TrackApplicationView` Higher order component exposed from the `usageCollection` plugin. You will need to add the plugin to plugin's `kibana.json` file as an item in the `optionalPlugins` and `requiredBundles` declarations:
```

Comment:
I added the phrases/sentences to describe what we're showing in the code snippet.